packages/api/src/entities/User.ts

```diff
@@ -10,9 +10,38 @@ export class User extends Node<User> {
   @Property({ columnType: 'text' })
   name: string;
 
-  constructor({ name, ...extraValues }: UserConstructorValues) {
+  @Property({ columnType: 'text' })
+  pronouns: string;
+
+  @Property({ columnType: 'text' })
+  location?: string;
+
+  @Property( { columnType: 'boolean'})
+  hireable?: boolean;
+
+  @Property({ columnType: 'text' })
+  purpose: string;
+
+  @Property({ columnType: 'text' })
+  schoolName:string;
+
+  @Property({ columnType: 'text' })
+  major: string;
+
+  @Property({ columnType: 'Date' })
+  graduationDate?: Date;
+
+
+  constructor({ name, pronouns, location, hireable, purpose, schoolName, major, graduationDate, ...extraValues }: UserConstructorValues) {
     super(extraValues);
 
     this.name = name;
+   
+    this.pronouns = pronouns;
+    this.purpose = purpose;
+    this.schoolName = schoolName;
+    this.major = major;
+    this.graduationDate = graduationDate;
+    // add other fields to constructor
```

Comment:
You only want to get the ones that are required, the optional ones will be set with `super(extraValues);` otherwise if they do supply them they would be `undefined` because nothing set them.

```suggestion
  constructor({ name, pronouns, hireable, purpose, ...extraValues }: UserConstructorValues) {
    super(extraValues);

    this.name = name;
    this.pronouns = pronouns;
    this.hireable = hireable;
    this.purpose = purpose;
```